MAUI/Segmented-Control/migration.md

```diff
@@ -0,0 +1,603 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI Segmented control (SfSegmentedControl) | Syncfusion
```

Comment:
title: Migrate from Xamarin to .NET MAUI Segmented Control (SfSegmentedControl) | Syncfusion

Reply:
title changed